sdk/core/aborter/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We should use the existing `sdk/.prettierrc.json`

Reply:
Removed `.prettierrc.json` from the `core/aborter` package and now using the one located under `sdk`. Ran `check-format` and `format` scripts and committed those changes as well.